src/oscar/templatetags/google_analytics_tags.py

```diff
@@ -0,0 +1,22 @@
+from django import template
+from django.templatetags.l10n import unlocalize
+
+
+register = template.Library()
+
+
+@register.filter()
+def make_gtag_item_list(order):
```

Comment:
You will need to add tests for any new template tags. It should be possible to do this directly in the template though, without the need for a tag?